src/components/Delegation/DelegatedToUserCard.tsx

```diff
@@ -0,0 +1,119 @@
+import ActionableLayout from '../Layout/ActionableLayout'
+import VotingPower from '../Token/VotingPower'
+import React, { useMemo, useState } from 'react'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
+import { Card } from 'decentraland-ui/dist/components/Card/Card'
+import { Stats } from 'decentraland-ui/dist/components/Stats/Stats'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { DelegationResult } from '../../hooks/useDelegation'
+import './DelegatedToUserCard.css'
+import VotingPowerListItem from './VotingPowerListItem'
+import { VotingPowerList } from './VotingPowerList'
```

Comment:
I'd rename this to VotingPowerListModal so we can detect easily this is a modal 🤔 What do you think?